cmd/lncli/cmd_pay.go

```diff
@@ -330,9 +335,27 @@ func sendPayment(ctx *cli.Context) error {
 		if err != nil {
 			return err
 		}
+		args = args.Tail()
 		req.FinalCltvDelta = int32(delta)
 	}
 
+	var payAddr []byte
+	switch {
+	case ctx.IsSet("pay_addr"):
+		payAddr, err = hex.DecodeString(ctx.String("pay_addr"))
+	case args.Present():
+		payAddr, err = hex.DecodeString(args.First())
+	}
+
+	if err != nil {
+		return err
+	}
+	if len(payAddr) != 32 {
```

Comment:
similar here, it's okay to not provide a `payAddr`

Reply:
Changed